### PR TITLE
Add CircleCI

### PR DESCRIPTION
**Pull request by viclim**
Fix https://github.com/GSA/i14y/issues/33

@nickmarden I created a docker image for elasticsearch with the necessary plugins installed. It's currently hosted in docker hub under my own account. You can create the image with `.circleci/elasticsearch/Dockerfile`, and host it using the official account.

### CODE REVIEW

**Comment by viclim** [comment]
CircleCI is not running.

**Comment by viclim** [comment]
TravisCI needs to be removed.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.3.6
+      - image: viclim/elasticsearch:i14y
```
Comment (nickmarden) [inline .circleci/config.yml]: Can you change this to simply `docker build -t elasticsearch:i14y` within the CI run? That way no hosting is required.

**Comment by nickmarden** [comment]
Travis CI is removed. Just move the Docker build into the `.circleci/config.yml` file and I think we're good to go.

**Comment by viclim** [comment]
CircleCI isn't running.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: cp -p config/secrets_example.yml config/secrets.yml
+      - run:
+          name: Install Docker client
+          command: |
+            set -x
+            VER="17.12.1-ce"
+            curl -L -o /tmp/docker-$VER.tgz https://download.docker.com/linux/static/stable/x86_64/docker-$VER.tgz
```
Comment (nickmarden) [inline .circleci/config.yml]: You don't need to install Docker - it's already available on CircleCI.

**Pull request merged**